DBM-Naxx/MilitaryQuarter/Horsemen.lua

```diff
@@ -41,6 +57,33 @@ function mod:SPELL_CAST_SUCCESS(args)
 	end
 end
 
+function mod:CHAT_MSG_MONSTER_YELL(msg)
+	if msg == "Death... will not stop me." or msg:find("Death... will not stop me.") then
```

Comment:
Let's keep localization separate as in:

https://github.com/SiarkowyForks/dbm-wotlk5man-lfr/blob/d0c2112fdd5404efd51dd01a202127efb87e9df4/DBM-Naxx/localization.en.lua#L408

You can register with `L:SetMiscLocalization({ Yell = ... })`, then access with `L.Yell`.

Reply:
Also at this point, I would suggest only using `str.find` (cause it includes equality check):
```suggestion
	if msg:find("Death... will not stop me.") then
```